test/test_n2.py

```diff
@@ -119,4 +109,4 @@
     e_hist[i, :] = int_e
 
 print(f"Exact energy: -109.10288938")
-print(f"Estimated energy: {np.min(e_hist[-1]) + nuclear_repulsion_energy}")
+print(f"Estimated energy: {np.min(e_hist[-1])}")
```

Comment:
Should this test have an assert statement?

Reply:
Yes I've been using this as a personal sanity check test, rather than something that runs in any automated way. I'm eventually going to try and write a Docker test that runs this in Ubuntu 24.04 in an automated way.

Here, we can remove this since we create an FCIDump from integrals only, so there is no notion of the nuclear energy anywhere in this workflow anymore. The user applies it at the outer layer, same as SQD